<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 0);</string>
  </property>
  <widget class="QPushButton" name="vendermas">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Seguir vendiendo</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconoProducto">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>3</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/agregarIcono2/IconoFarmacia2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="terminar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tab_ventas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>111</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
